spec/forms/provider_interface/offer_wizard_spec.rb

```diff
@@ -39,10 +44,6 @@
     it { is_expected.to validate_presence_of(:course_option_id).on(:locations).on(:save) }
     it { is_expected.to validate_presence_of(:study_mode).on(:study_modes).on(:save) }
     it { is_expected.to validate_presence_of(:course_id).on(:courses).on(:save) }
-    it { is_expected.to validate_length_of(:further_condition_1).is_at_most(255) }
```

Comment:
can we add a test that we surface the validations that come from the offer conditions object?

Reply:
Yes, also found a bug with this!